README.md

```diff
@@ -34,6 +34,10 @@ You can also install these dependencies via pip (for Python 3) using: `pip3 inst
             497939890063802369:
                 name: ovddsc
                 use_webhooks: false
+                # Set this value to configure relay for messages sent via non-pylink webhooks 
+                webhooks_agent: false # disables webhook relaying
+                webhooks_agent: Webhooks # specifies nick
+                webhooks_agent: Webhooks!webhooks@discord/webhooks-agent # specifies nick, host and ident
```

Comment:
I realize that my description in chat was very brief. This would be more adequately documented as:

```yaml
# Sets the name of the dummy client used to forward webhooks from non-PyLink sources,
# since they don't correspond to Discord users. Valid values include false (disable webhook
# forwarding entirely), a nick, or a full hostmask (nick!user@host).
# This defaults to false if not set.
webhooks_agent: false
#webhooks_agent: Webhooks
#webhooks_agent: Webhooks!webhooks@discord/webhooks-agent
```

The other examples should be commented out, since only one value can be active at a time.